front/src/common/BootstrapSNCF/InputGroupSNCF.tsx

```diff
@@ -29,7 +29,7 @@ type Props = {
   min?: number;
   max?: number;
   step?: number | string;
-  textRight?: boolean;
+  textAlign?: 'left' | 'right' | 'center';
```

Comment:
Is "center" a wise choice for an input ? i think left and right were sufficient.
Plus left is the native behaviour of alignement so you should have kept it with "textRight" props to handle the only case where it's not left

Reply:
I think that was a request from @nicolaswurtz

Reply:
Nope, textRight should be false by default.

Reply:
Done